### DISCUSSION TITLE
How to set somes system as training set?

### DISCUSSION THREAD

**Discussion by Xi-yuanWang** [deepmodeling/deepmd-kit#264] [Q&A]
It seems that deepmd-kit uses certain sets in a system as test sets. Then, if one uses a large amount of system data, how to set some systems as the training set.

**Comment by njzjz** [comment]
deepmd-kit uses the last set as the test set, if the number of sets is more than 1. You can assign one of your test sets as the last set, and remove others.

**Comment by Xi-yuanWang** [comment]
Thanks. But different systems have different type.raw. So I just divide it by myself. And now I'm wondering how to use dp test to test various systems.

**Reply by akshata-pixel** [reply]
I also have a same question...
